apps/api/src/domain/models/term/commands/add-audio-for-term/add-audio-for-term.command.ts

```diff
@@ -0,0 +1,25 @@
+import { AggregateType, ICommandBase } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, ReferenceTo, UUID } from '@coscrad/data-types';
+import { AggregateId } from '../../../../types/AggregateId';
+import { TermCompositeIdentifier } from '../create-term';
+
+@Command({
+    type: 'ADD_AUDIO_FOR_TERM',
+    label: 'add audio for term',
+    description: 'add audio for an existing term',
+})
+export class AddAudioForTerm implements ICommandBase {
+    @NestedDataType(TermCompositeIdentifier, {
+        label: 'term composite identifier',
+        description: 'system-wide unique identifier',
+    })
+    readonly aggregateCompositeIdentifier: TermCompositeIdentifier;
+
+    @UUID({
+        label: 'audio item ID',
+        description: 'reference to an audio item',
+    })
+    @ReferenceTo(AggregateType.audioItem)
+    readonly audioItemId: AggregateId;
```

Comment:
We need to add a `languageCode` here because in the future you'll be able to add audio for any of the languages via a `MultilingualAudio`.

Reply:
done